doc/example/basic.rst

```diff
@@ -59,6 +59,30 @@ The environment ``py`` uses the version of Python used to invoke tox.
 However, you can also create your own test environment names,
 see some of the examples in :doc:`examples <../examples>`.
 
+pyproject.toml tox legacy ini
+-----------------------------
+
+It's possible to put the tox ini configuration into the ``pyproject.toml`` file too (if want to avoid
+an extra file):
+
+.. code-block:: toml
+
+   [build-system]
+   requires = [ "setuptools >= 35.0.2", "wheel >= 0.29.0"]
+   build-backend = "setuptools.build_meta"
+
+   [tool.tox]
+   legacy_tox_ini = """
+   [tox]
+   envlist = py27,py36
+
+   [testenv]
+   deps = pytest >= 3.0.0, <4
+   commands = pytest
+   """
```

Comment:
this seems not worth it :man_shrugging:  -- we should probably wait until we have a toml-specific parse / structure?

Reply:
See #814; we've agreed as first step this is worthy. We'll move to a full toml-specific parser as step 2. This allows a quick move over for people wanting to do so.

Reply:
ah got it :+1: